misc/config_tools/upgrade_scenario_xml

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+function usage() {
+	echo "Usage: $0 <scenario XML> [<version of the given scenario XML>]"
+	echo "Upgrade the given scenario XML to align with the current version of ACRN."
+	echo ""
+	echo "Example: $0 logical_partition.xml v2.3"
+	echo "    The output file is logical_partition-v2.3.xml"
+}
+
+file=$1
+ver=$2
+
+if [[ "${file}" = "" ]]; then
+	usage
+	exit 1
+fi
+
+if [[ ! -f "${file}" ]]; then
+	echo "${file}: No such file"
```

Comment:
Probably no harm in displaying here also the `usage()`

Reply:
Sure